test/compliance/INSERT.test.js

```diff
@@ -32,4 +36,13 @@ describe('INSERT', () => {
       throw new Error('not supported')
     })
   })
+
+  describe('default values', () => {
+    test('default values are generated', async () => {
+      const { 'basic.literals.defaults': entity } = cds.db.entities
+      await cds.run(INSERT.into(entity).entries({ID: 1}))
+      const result = await cds.run(SELECT.from(entity, 1))
+      assert.deepEqual(result, { ID: 1, boolean: false, integer: 0, nulls: null, string: ''})
```

Comment:
I also like `assert`, but I do think it would be best to make it work with `expect` for overall consistency.

Reply:
we do use `assert` already quite heavily, don't we? I changed from `expect` to `assert` as we want to go towards `assert` in the future?